libcontainerd/remote/client.go

```diff
@@ -217,12 +216,6 @@ func (c *client) Start(ctx context.Context, id, checkpointDir string, withStdin
 				opts.IoUid = uint32(uid)
 				opts.IoGid = uint32(gid)
 				info.Options = &opts
-			} else {
-				info.Options = &runctypes.CreateOptions{
-					IoUid:       uint32(uid),
-					IoGid:       uint32(gid),
-					NoPivotRoot: os.Getenv("DOCKER_RAMDISK") != "",
```

Comment:
Not at my computer right now, but do we actually set this option based on this env-var elsewhere for v2? I found it in the builder code, but at a glance didn't see it for other code 🤔 https://github.com/moby/moby/blob/0a3336fd7d19f7114fce2ff849a8989ed33e2059/builder/builder-next/executor_unix.go#L59

(I can check when I'm at my computer)

Reply:
Yes
https://github.com/moby/moby/blob/085c6a98d54720e70b28354ccec6da9b1b9e7fcf/daemon/runtime_unix.go#L43-L53